src/metaxy/metadata_store/ibis.py

```diff
@@ -206,6 +236,27 @@ def _validate_hash_algorithm_support(self) -> None:
                 f"Supported algorithms: {', '.join(supported)}"
             )
 
+    def _ensure_system_tables_exist(self) -> None:
```

Comment:
This should be dropped, the normal metadata write method already creates tables if auto creation is enabled